feature/policy_forwarding/encapsulation/otg_tests/static_encap_gue_ipv4/README.md

```diff
@@ -0,0 +1,334 @@
+
+*   DUT and DUM have 5 ports each.
+
+    ```
+                              -------                  -------
+                             |       | ==== LAG1 ==== |       |
+          [ ATE:Port1 ] ---- |  DUT  |                |  DUM  | ---- [ ATE:Port2 ]
+                             |       | ==== LAG2 ==== |       |
+                              -------                  -------
+    ```
+
+*   Routes are advertised from ATE:Port2.
+*   Traffic is generated from ATE:Port1.
+*   ATE:Port2 is used as the destination port for GUE encapsulated traffic.
+
+#### Configuration
+
+1.  DUT:Port1 is configured as Singleton IP interface towards ATE:Port1.
+
+2.  DUT:Port2 and DUT:Port3 are configured as LAG1 IP interface towards
+    DUM:Port2 and DUM:Port3 respectively.
+
+3.  DUT:Port4 and DUT:Port5 are configured as LAG2 IP interface towards
+    DUM:Port4 and DUM:Port5 respectively.
+
+4.  DUM:Port1 is configured as Singleton IP interface towards ATE:Port2.
+
+5.  DUT is configured to form two eBGP sessions with DUM using the directly
+    connected LAG interface IPs.
+
+6.  DUM is configured to form an eBGP sessions with ATE:Port2 using the directly
+    connected Singleton interface IP.
+
+7.  ATE:Port2 is configured to advertise destination networks (IPv4-DST-NET,
+    IPv6-DST-NET) and tunnel destination (IPv4-DST-GUE) to DUM. When DUM
+    advertised these prefixes to the DUT over the two eBGP sessions, the
+    protocol next hops for the destination networks should be re-configured as
+    below:
+
+    -   Destination network IPv4-DST-NET with protocol next hop PNH-IPv4.
+    -   Destination network IPv6-DST-NET with protocol next hop PNH-IPv6.
+
+8.  DUT is configured with an IPv4 GUE tunnel with destination IPv4-DST-GUE
+    without any explicit tunnel Type of Service (ToS) or Time to Live (TTL)
+    values.
+
+9.  DUT is configured with the following static routes:
+
+    -   To PNH-IPv4, next hop is the statically configured IPv4 GUE tunnel.
+    -   To PNH-IPv6, next hop is the statically configured IPv4 GUE tunnel.
+
+### PF-1.3.1: IPv4 traffic GUE encapsulation without explicit ToS/TTL configuration on tunnel
+
+ATE action:
+
+*   Generate **IPv4 traffic** from ATE:Port1 to random IP addresses in
+    IPv4-DST-NET using a random combination source addresses at linerate.
+    *   Use 512 bytes frame size.
+    *   Set ToS value of *0x80* for all packets.
+    *   Set TTL of the packets to *10*.
+
+Verify:
+
+*   Policy forwarding packet counters matches the packet count of traffic
+    generated from ATE:Port1.
+*   The packet count of traffic sent from ATE:Port1 should be equal to the sum
+    of all packets egressing DUT ports 2 to 5.
+*   All packets egressing DUT ports 2 to 5 are GUE encapsulated.
+*   ECMP hashing works (equal traffic) over the two LAG interfaces.
+*   LAG hashing works (equal traffic) over the two Singleton ports.
+*   ToS for all GUE encapsulated packets:
+    *   GUE header ToS is **0x80**.
```

Comment:
Can we perhaps clarify throughout what GUE header means. In the draft (`draft-ietf-intarea-gue-09`) then the GUE header is defined to be a header on top of the UDP header (see the diagram). I think that maybe clarifying somewhere at the start of this README that this is GUE variant 1, which can be interpreted as IPoUDPoIP is perhaps worth doing so that we clearly state whether the GUE header as defined by the RFC is needed.

Reply:
Yes, added details to the beginning of the README to clarify.